scripts/jenkins/jobs-ibs/cloud-mkcloud6-scenario-dvr.yaml

```diff
@@ -0,0 +1,25 @@
+- job:
+    name: 'cloud-mkcloud6-scenario-dvr'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H 19 */2 * *'
```

Comment:
you mean why its running every 2nd day instead of every day? Its just similar to how the other scenarios are configured. we can run it daily if you want.